### PR TITLE
Implement the Open Locally button on file previews [SCI-9757]

### PR DESCRIPTION
**Pull request by G-Chubinidze**
Jira ticket: [SCI-9757](https://scinote.atlassian.net/browse/SCI-9757)

### What was done
_implement open locally button for file previews_

[SCI-9757]: https://scinote.atlassian.net/browse/SCI-9757?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: app/javascript/packs/vue/open_locally_menu.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+/* global notTurbolinksPreview */
```
[the remark this answers is not in the document]
Comment (aignatov-bio) [reply app/javascript/packs/vue/open_locally_menu.js:1]: Why this here?

File: app/javascript/vue/shared/content/attachments/open_locally_menu.vue [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+
+        if (this.attachment.wopi && this.attachment.urls.edit_asset) {
+          menu.push({
+            text: this.attachment.wopi_context.button_text,
+            url: this.attachment.urls.edit_asset,
+            url_target: '_blank'
+          })
+        }
+        if (this.canOpenLocally) {
+            console.log(this.localAppName)
```
Comment (aignatov-bio) [inline app/javascript/vue/shared/content/attachments/open_locally_menu.vue:51]: Remove

File: app/views/shared/file_preview/_content.html.erb [hunk 3]
```diff
@@ -3,13 +3,9 @@
...
       <% if wopi_enabled? && wopi_file?(asset) %>
-        <% edit_supported, title = wopi_file_edit_button_status(asset) %>
-        <%= render partial: 'assets/wopi/file_wopi_controls',
-          locals: {
-            asset: asset,
-            edit_supported: edit_supported,
-            title: title
-          } %>
+        <div id="openLocallyMenu" data-behaviour="vue">
+          <open-locally-menu data="<%= AssetSerializer.new(asset, scope: { user: current_user }).to_json %>"/>
```
Comment (aignatov-bio) [inline app/views/shared/file_preview/_content.html.erb:7]: this must be done with remote request. Just add show action to AssetsController

**Review by aignatov-bio — CHANGES_REQUESTED** [review]
Fix hound

**Review by aignatov-bio — CHANGES_REQUESTED** [review]
Fix hound

**Review by artoscinote — APPROVED** [review]

**Pull request merged**